src/cio_websocket.c

```diff
@@ -421,7 +436,11 @@ static void get_mask(struct cio_buffered_stream *bs, void *handler_context, enum
 
 	memcpy(ws->mask, ptr, sizeof(ws->mask));
 	if (likely(ws->read_frame_length > 0)) {
-		bs->read_exactly(bs, buffer, ws->read_frame_length, get_payload, ws);
+		err = bs->read_exactly(bs, buffer, ws->read_frame_length, get_payload, ws);
+		if (unlikely(err != CIO_SUCCESS)) {
+			handle_error(ws, CIO_WEBSOCKET_CLOSE_TOO_LARGE, "payload length too large to handle");
+			return;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant jump. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS3626)